dashboard/src/components/workspace/WorkspaceHeader.tsx

```diff
@@ -112,9 +106,23 @@ export default function WorkspaceHeader() {
               >
                 <Dropdown.Item
                   className="py-2"
-                  onClick={() =>
-                    setChangeWorkspaceNameModal(!changeWorkspaceNameModal)
-                  }
+                  onClick={() => {
+                    openDialog('EDIT_WORKSPACE_NAME', {
+                      title: (
+                        <span className="grid grid-flow-row">
+                          <span>Change Workspace Name</span>
+                          <Text variant="subtitle1" component="span">
+                            Changing the workspace name will also affect the URL
```

Comment:
Check copy & double-check that we want to also change the slug

Reply:
I think we should change the slug if we are doing it for projects as well. But maybe @nunopato / @elitan has a better answer to this.

Reply:
> change the slug if we are doing it for projects as well.

Yep, that makes sense. Then we are good.